xcixor/ireporter-api
# Chore Set up database

## What is the Purpose of this PR
The purpose of this PR is to set up DB API
## Why is it important
To ensure permanent storage of data
## Tasks completed
-Add app configurations
-Add app dependencies
-Add app initialization and run scripts
-Add database set up script
-Add app installation to readme.md
## How can it be tested 
- clone the repo [@here](https://github.com/xcixor/ireporter-api) 
- switch to the branch chore-setup-db-162475977
- Follow the instructions in README.md on how to setup app
- run the application through
$ python run.py
The logs in the terminal should print "***** db init complete *****"

## Relevant PT story
[#162475977](https://www.pivotaltracker.com/story/show/162475977)
